demos/django/docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+# Include syntax requires Docker compose > 2.20.3
+# https://docs.docker.com/compose/release-notes/#2203
+include:
+  # Creates a standard Postgress instance
+  - path: ../../services/postgres.yaml
+
+  # Creates the internal MongoDB replica set
+  - path: ../../services/mongo.yaml
+
+  # Main PowerSync backend
+  - path: ../../services/powersync.yaml
+
+  # Demo NodeJS backend server and front-end web client
+  - path: ../../demos/django/ps-django-backend.yaml
```

Comment:
Can this be just `./ps-django-backend.yaml`?

Comment is also wrong

Reply:
Good catch! I'm not sure how it works with both `path`s. I've updated this now.